include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -553,26 +573,33 @@ class argument_parser
             return;
         }
 
+        bool special_format_was_set{false};
```

Comment:
Still don't understand why you need this. Just return as before.

Reply:
I need to scan the whole argv in order to strip the version check option from the argv if present at any position. I cannot just return at any place.

Reply:
Hm, it's ugly, but oh well.